### ISSUE TITLE
Table readability

### ISSUE THREAD

**Issue by dataf3l** [cblp/yaml-sucks#7]
When I read the table I didn't realise there were 5 columns, I only saw 2 columns, perhaps there is a way to solve this issue, somehow (linking to an external html table perhaps?)

**Comment by cblp** [comment]
Yes, this would be nice, I'll do it sometime. You can also read http://matrix.yaml.io/

**Comment by Jarzon** [comment]
I tried to make it more readable by making a table for each YAML source: https://github.com/Jarzon/yaml-sucks/tree/readable-table

**Comment by ENBYSS** [comment]
> 
> 
> I tried to make it more readable by making a table for each YAML source: https://github.com/Jarzon/yaml-sucks/tree/readable-table

Your table seems to be broken for `rq` - every entry is now replaced with `rq: command not found`.

**Comment by Jarzon** [comment]
It's been fixed in #11 and I didn't update since then.

**Comment by xjcl** [comment]
**Table readability is extremely important**, if you want the reader to actually critically think about the specific examples.

Maybe try making a separate `github.io` page (which has more width)? Other than that I guess you could separate the `{"a":{"k2":"v2","k1":"v1"},"b":{"k3":"v3","k2":"v2","k1":"override"}}` onto multiple lines, but even that only yields about 3 columns, not 5.

I would be for merging @Jarzon's repo back.

**Comment by shoogle** [comment]
> I only saw 2 columns

Same, but I think there's an easy solution. Simply edit the README and add this directly above the table:

__>>>> Scroll to the right to see more implementations! >>>>>__
